src/core_plugins/timelion/server/series_functions/aggregate/first.js

```diff
@@ -0,0 +1,5 @@
+import _ from 'lodash';
+
+module.exports = function (points) {
+  return _.first(points);
```

Comment:
do we really need a wrapper around lodash ? we use it directly everywhere else in our codebase ? (goes for this, min, max, sum, last)